app/api/dds/v1/app_api.rb

```diff
@@ -55,33 +51,36 @@ class AppAPI < Grape::API
             status[:graphdb] = 'ok'
           else
             status[:status] = 'error'
-            status[:graphdb] = 'environment is not set'
+            logger.error 'graphdb environment is not set'
           end
 
           if ENV['CLOUDAMQP_URL']
-            missing_exchanges = [
-              ApplicationJob.opts[:exchange],
+            [
+              Sneakers::CONFIG[:exchange],
+              Sneakers::CONFIG[:retry_error_exchange],
               ApplicationJob.distributor_exchange_name
-            ].reject do |expected_exchange|
-              ApplicationJob.conn.exchange_exists? expected_exchange
+            ].each do |expected_exchange|
+              unless ApplicationJob.conn.exchange_exists? expected_exchange
```

Comment:
We need to change this to use something other than the (private) '.conn' method. A future PR will allow the usage of Sneakers::CONFIG[:connection] for this purpose.